generate_postmortem.sh

```diff
@@ -9,6 +9,10 @@
 #
 
 #parse passed arguments
+PMCOMMIT='33f111daa2f772e18d43246510440212de0c03f7'
+PMCOMMITURL="https://github.com/ibm-apiconnect/v10-postmortem/commit/$PMCOMMIT"
+echo "Postmortem Version: $PMCOMMITURL"
```

Comment:
Maybe move this line after "switch case" section, so that when running --version we only get one line not 2.
We could remove this line entirely, but I kinda like that even if the script fails to complete and the customer can only send some screenshot of his shell window, that we will be able to see which version they are using.